terraform/gitops/generate-files/templates/mojaloop/values-finance-portal.yaml.tpl

```diff
@@ -126,6 +126,8 @@ reporting-hub-bop-shell:
       REMOTE_API_BASE_URL: ''
       REMOTE_MOCK_API: false
       LOGIN_URL: https://${auth_fqdn}/kratos/self-service/login/browser
+      ## return_url can be passed as query param to LOGOUT_URL for redirect the user back to portal after logout
+      ## Example: return_url=http%3A%2F%2F${keycloak_fqdn}%2Frealms%2F${keycloak_realm_name}%2Fprotocol%2Fopenid-connect%2Flogout%3Fclient_id%3D${hubop_oidc_client_id}%26post_logout_redirect_uri%3Dhttps%3A%2F%2F${portal_fqdn}
```

Comment:
It must be passed, we should not have a default one, as it is specific to a realm.

Reply:
Yes you are right, let me pass it in

Reply:
I passed the return url with realm. And I still hold on using client_id in the redirect uri and placed a comment so that we can enable if we want. Lets check if the current approach works. It just ends up on keycloak logout page and user need to open the portal manually if he wants to login again.